packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -537,6 +537,13 @@ export interface ProjectProps extends CommonProjectProps {
    */
   readonly secondarySources?: ISource[];
 
+  /**
+   * The secondary source versions for the Project.   *
+   * @default - No secondary source versions (they default to latest).
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html#cfn-codebuild-project-sourceversion
+   */
+  readonly secondarySourceVersions?: CfnProject.ProjectSourceVersionProperty[];
```

Comment:
This should not be here, right? It's not actually used anywhere.

Every `ISource` passed in `secondarySources` has their own `sourceVersion` property returned from `bind`, and that's what should be filled for the `secondarySourceVersions` property of `CfnProject`.

Also, remember that we never use L1 types (those that start with `Cfn*`) for L2 properties.

Please remove this property.